src/ctap/data_formats.rs

```diff
@@ -1322,6 +1338,17 @@ mod test {
         assert_eq!(sub_command, Ok(expected_sub_command));
         let created_cbor: cbor::Value = sub_command.unwrap().into();
         assert_eq!(created_cbor, cbor_sub_command);
+
+        #[cfg(not(feature = "with_ctap2_1"))]
+        let last_literal = 0x05;
+        #[cfg(feature = "with_ctap2_1")]
+        let last_literal = 0x09;
```

Comment:
We would not need this using the `DoNotUseAndKeepLast` enum value. It would automatically get the right value.